storage/spider/spd_db_mysql.cc

```diff
@@ -8069,15 +8068,6 @@ int spider_mbase_handler::append_insert(
       DBUG_RETURN(HA_ERR_OUT_OF_MEM);
     str->q_append(SPIDER_SQL_LOW_PRIORITY_STR, SPIDER_SQL_LOW_PRIORITY_LEN);
   }
-  else if (spider->insert_delayed)
-  {
-    if (share->internal_delayed)
```

Comment:
Since `share->internal_delayed` is only used here, it would not be a problem to delete `internal_delayed` itself from `SPIDER_SHARE`. However, since it is one of setting value of the remote table, I could not decide whether it is OK to delete it just because it is no longer used.

Reply:
Thank you for pointing this out. The table parameter `internal_delayed` could be considered as a part of the delayed insert support. So, I believe we could delete this. https://mariadb.com/kb/en/spider-table-system-variables/

Could you delete the parameter and the corresponding member of `SPIDER_SHARE` and then mention the fact that we deleted it in the commit message?